src/app/app-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [];
+
+@NgModule({
+  imports: [RouterModule.forRoot(routes)],
+  exports: [RouterModule]
+})
+export class AppRoutingModule { }
```

Comment:
Routes is empty, you can even remove this file if you don't think to implement routing, but I think that it would be a good feature to have the routing per search/page